api/src/pcapi/connectors/typeform.py

```diff
@@ -143,7 +142,19 @@ def get_form(self, form_id: str) -> TypeformForm:
     def get_responses(
         self, form_id: str, num_results: int = 100, sort: str = "submitted_at,desc", since: datetime | None = None
     ) -> list[TypeformResponse]:
-        return []
+        return [
+            TypeformResponse(
+                response_id="",
```

Comment:
Vraisemblablement ça ne marche qu'au premier import, il faut un id différent par réponse.
```suggestion
                response_id=f"{form_id}-response",
```